### DISCUSSION TITLE
Importing modules

### DISCUSSION THREAD

**Discussion by Jimpex** [dpmland/dpm#4] [Q&A] [answered]
I am new to deno and dpm and I can't find any documentation anywhere on usage. How do I import a module I installed into my file and use it?

**Comment by TeoDev1611** [comment, marked as answer]
Hello!

Currently I am working in the development branch with the documentation and can you install the development branch with the news and documentation with this: 

```
deno run -A --unstable https://denopkg.com/dpmland/dpm@dev/install.ts
```

And for use the modules imported in the import map you need use like node:

```ts
import { serve } from "http/server.ts";
serve(req => new Response("Hello World\n"));
```

I hope help you thanks for use DPM :)
